Added comms lead role notes (#346)
alphagov/paas-team-manual: source/team/comms_lead_role.html.md

# Comms lead role - PaaS Incidents

### What to do first:
Learn about [Pagerduty](/team/pagerduty/)

1. Pagerduty calls/alerts you
1. Go on Slack - PaaS incident channel (and PaaS internal channel)

> 1. Set up a hangout for you and the engineer (add tenant only if needed+useful)

Reviewer: ```suggestion
1. Set up a hangout for you and the engineer (add tenant only if it will be useful)
```

> 1. Make a copy of the [incident report](https://docs.google.com/document/d/1w_16Z__z5LY8UCpxjtRjoXimQHP1WhRdEq70nzsYprI/edit) and start to fill it in

Reviewer: ```suggestion
1. Make a copy of the [incident report](https://docs.google.com/document/d/1w_16Z__z5LY8UCpxjtRjoXimQHP1WhRdEq70nzsYprI/edit), name the document and start to fill it in
```

> 1. Record timeline, and whats happening as you go in the incident report

Reviewer: ```suggestion
1. Record the timeline of events as they happen in the incident report
```

> ### Statuspage - creating incident:

Reviewer: ```suggestion
### Statuspage - creating an incident:
```

Learn about [Statuspage](/team/statuspage/)

> 1. [StatusPage.io](https://www.statuspage.io/) - go in and create incident (name it)

Reviewer: ```suggestion
1. [StatusPage.io](https://www.statuspage.io/) - log in and create an incident (name it)
```

> 1. Select apply template (possible issue being investigated/we’re having an incident etc)

Reviewer: ```suggestion
1. Select apply template (for example, possible issue being investigated/we’re having an incident)
```

> 1. Select apply template (possible issue being investigated/we’re having an incident etc)

Reviewer: ```suggestion
1. Select 'apply template' (possible issue being investigated/we’re having an incident etc)
```

1. Fill in the template with relevant details

> 1. You can choose components that are affected

Reviewer: ```suggestion
1. You can choose the components (for example, API, apps, billing) that are affected
```

> 1. Select send notifications (to email tenants). If it’s a small issue, you may not want to send them a notification (then status page will only be updated)

Reviewer: ```suggestion
1. Select 'send notifications' to email tenants. If it’s a small issue, you may not want to send them a notification (in this case, only our status page will be updated)
```

1. The subscribers on statuspage will get the notifications

### If you need to escalate to SMT on call:

> 1. If you need to escalate to SMT (e.g. if its affecting coronavirus services) - go to [rotas app](https://rotas.cloudapps.digital/teams/techops-management-escalations) and click on the current on call individual to get their contact info 

Reviewer: ```suggestion
1. If you need to escalate to SMT (for example, if its affecting coronavirus services) - go to [rotas app](https://rotas.cloudapps.digital/teams/techops-management-escalations) and click on the current on call individual to get their contact info 
```

### Don’t forget:

> 1. Aim is to do just enough support out of hours to get through to working hours :)

Reviewer: ```suggestion
1. Your aim is to do just enough support out of hours to get through to working hours :)
```

> 1. You can do an update on x-gov slack paas channel (if relevant)

Reviewer: ```suggestion
1. You can update the x-gov slack paas channel if relevant
```
